src/app/analytics/analytics.factory.js

```diff
@@ -187,18 +187,16 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
         // Error caught in analyticsFactory.cartRemove
       }
     },
-    cartView: function (cartData, callType) {
+    cartView: function (isMiniCart = false) {
       try {
-        // Build products variable
-        this.buildProductVar(cartData)
         // Send GTM Advance Ecommerce event
         if (typeof $window.dataLayer !== 'undefined') {
           $window.dataLayer.push({
-            event: callType === 'mini-cart' ? 'view-mini-cart' : 'view-cart'
+            event: isMiniCart ? 'view-mini-cart' : 'view-cart'
           })
         }
         // Call DTM direct call rule
-        if (typeof callType !== 'undefined' && callType === 'customLink') {
+        if (isMiniCart) {
```

Comment:
This block doesn't really make sense.

Reply:
Yeah, I'm not sure what the logic before was, but the event being fired is an ```aa-view-minicart``` event. So I figured we would only want that to be called if ```isMiniCart``` is true?

Reply:
It was never called, so why not just remove the block?

Reply:
Ah good point, yep will do!